### PR TITLE
Added RHEL9 Support for installation

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1- Extened RHEL8 condition to RHEL9 as kick start need BaseOS as repo 
2- setup root login on RHEL9

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Review by abdhaleegit — APPROVED** [review]
few comments

**Comment by abdhaleegit** [comment]
it stuck at here 
```
in kickstart2:34:26,802 [DEBUG]  
2023-02-01 22:34:42,961 [DEBUG]  Starting automated install......Saving storage configuration...
2023-02-01 22:34:44,089 [DEBUG]  .Checking storage configuration...
2023-02-01 22:34:45,235 [DEBUG]  .
2023-02-01 22:34:46,752 [DEBUG]  .================================================================================ 22:34:26,803 [DEBUG]  Question
2023-02-01 22:34:46,752 [DEBUG]  =======================================================================         Problems in request:4:28,400 [DEBUG]  [anaconda]1:main* 2:shell  3:log  4:sto><'echo -n "Switch
2023-02-01 22:34:46,753 [DEBUG]  missing packages: java, telnet
2023-02-01 22:34:46,753 [DEBUG]  
2023-02-01 22:34:46,753 [DEBUG]  Would you like to ignore this and continue with installation?
2023-02-01 22:34:46,753 [DEBUG]  
2023-02-01 22:34:46,753 [DEBUG]  
```

File: installvm.py [hunk 1]
```diff
@@ -321,21 +321,24 @@ def createKickstart(self):
                 host_disk += '/dev/disk/by-id/' + disk+','
             vmParser.args.host_disk = host_disk.rstrip(',')
 
-        if '8' not in version:
+        if version.startswith('8') or  version.startswith('9'):
             lstr = "telnet\njava\n%end"
```
Comment (abdhaleegit) [inline installvm.py:325]: this line should go to else `lstr = "telnet\njava\n%end"`

Comment (PraveenPenguin) [reply installvm.py:325]: thanks for review , yes will adopt

**Pull request merged**